prow/scripts/cluster-integration/reconciler-e2e-gardener.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+
+#Description: Kyma CLI Integration plan on Gardener. This scripts implements a pipeline that consists of many steps. The purpose is to install and test Kyma using the CLI on a real Gardener cluster.
+#
+#Expected common vars:
+# - JOB_TYPE - set up by prow (presubmit, postsubmit, periodic)
+# - KYMA_PROJECT_DIR - directory path with Kyma sources to use for installation
+# - GARDENER_REGION - Gardener compute region
+# - GARDENER_ZONES - Gardener compute zones inside the region
+# - GARDENER_CLUSTER_VERSION - Version of the Kubernetes cluster
+# - GARDENER_KYMA_PROW_KUBECONFIG - Kubeconfig of the Gardener service account
+# - GARDENER_KYMA_PROW_PROJECT_NAME - Name of the gardener project where the cluster will be integrated.
+# - GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME - Name of the secret configured in the gardener project to access the cloud provider
+# - MACHINE_TYPE - (optional) machine type
+#
+#Please look in each provider script for provider specific requirements
+
+
+# exit on error, and raise error when variable is not set when used
+set -e
+
+ENABLE_TEST_LOG_COLLECTOR=false
+
+export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+export RECONCILER_SOURCES_DIR="/home/prow/go/src/github.com/kyma-incubator/reconciler"
+export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
+#set to dummy value, required by gardener/gcp.sh
+export KYMA_SOURCE="main"
```

Comment:
The `KYMA_SOURCE` variable is unused, should we remove it?

Reply:
removed